src/issues/issue-2/wedding/styled.js

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components"
+import { mediaQueries } from "../../../styles/layout"
+
+export const Wrapper = styled.div`
```

Comment:
I would name this `Background`, as Wrapper is a little too generic, usually, you should prepend `wrapper` with a descriptor of what you're wrapping, e.g, `ImgWrapper`, `CarouselWrapper`, `CardWrapper` etc